### PR TITLE
U/craiglagegit/daytime checkout

### PR DESCRIPTION
**Pull request by craiglagegit**
New DayTime-Checkout with cahnges for Cycle 25 software.

### CODE REVIEW

File: procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb [hunk 1]
```diff
@@ -5,7 +5,7 @@
    "id": "3dbf3c0c",
```
Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Modified for ATOODS
</pre><p>you can remove this comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='19'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Modified for ATOODS
</pre><p>you can remove this comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='22'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>Add some context here for why this happens.</p><p>The current DDS configuration results in history being lost when CSCs are restarted and/or a small network outage occurs. If this happens, then to receive the summaryState event, which is required by salobj.get_summary_state (and used extensively in the atcs), it is necessary to manually change the state of a CSC. The commands below can be used to perform this action.</p><p>A new DDS configuration is now ready to be tested and will hopefully be released in the near future.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>Seeing as these are not be done regularly, and they'll be removed in the future, please leave them commented.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>I don't think this is necessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>you've seen it fault here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>Do you need this line? can't you just go to enabled?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='41'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>Why is this here? If required, please add a line stating the reasoning.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='61'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Modified for ATOODS
</pre><p>you can remove the comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='80'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Modified for ATOODS
</pre><p>here too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='84'/>

Comment (craiglagegit) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: OK. Removed.

Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>Moved these commands to a Troubleshooting Section at the end (specially since this condition will soon disappear)</p>

Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>Moved to the end <code>Quick Troubleshooting</code> section </p>

Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>Deleted since the dome motion is tested in previous cell with </p><p><code>await atcs.rem.atdome.cmd_moveAzimuth.set_start(azimuth=dome_az.azimuthPosition+d_az)</code></p><p> </p>

File: procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb [hunk 2]
```diff
@@ -120,90 +120,11 @@
...
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "0b014428-5475-4530-8c42-2b969c315bf6",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# All up now"
+    "If the camera and/or archiver don't come up, see section `Quick Troubleshooting` for support commands. "
```
Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb]: should probably replace archiver with atoods since the csc was (sort of) renamed.

File: procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb [hunk 3]
```diff
@@ -5,10 +5,10 @@
    "id": "3dbf3c0c",
```
Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>maybe change archiver to atoods since it was (sort of) renamed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='10'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>can you add a rot_type=RotType.PhysicalSky here? Then we don't have to wait for the rotator to slew really far.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='38'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>Optional, but might be nice to make this a markdown heading? I think with 4 #'s ? The ones below as well...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='72'/>

File: procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb (hunk 2 above)
Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb]: Completely right!

File: procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb (hunk 3 above)
Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>Sounds good. </p>

Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>Thank you!. I used ### instead of 4 for the structure of the Table of Contents</p>

Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:5]: <p>Yes!</p>

**Review by patrickingraham — APPROVED** [review]
Thanks for making the changes.

**Comment by patrickingraham** [comment]
Please rebase and squash your commits before merging. Let me know if you need help.
